libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -149,3 +149,56 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
                 )
                 for page in doc.pages
             ]
+
+
+class AmazonTextractPDFParser(BaseBlobParser):
+    """sends PDF files to  Amazon Textract and parses them to generate Documents.
+    multi-page documents have to reside on S3, single page PDF can be on local file system
+    multi-page documents that do not reside on S3 will throw an ValueError
+    """
+
+    try:
+        import textractcaller as tc
```

Comment:
Why are we using the textractcaller package, and not boto3 directly?

Reply:
Because the textractcaller abstract the complexity of calling Textract. Otherwise the code would be quite long including polling for the async response.